Makefile

```diff
@@ -8,3 +8,12 @@ upgrade-requirements:
 
 docs:
 	cd docs && $(MAKE) html; cd -
+
+migrations: TMPLOCK:=migrations.new.lock
+migrations:
```

Comment:
based on the name I would expect `make migrations` to do something similar to `./manage.py makemigrations`. How about renaming this to `migrations_lock` or `migrations_list`?

Reply:
+1 for `make migrations.lock` (if dots are allowed) if not, `make migrations_lock`